stdlib/either.ml

```diff
@@ -0,0 +1,66 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*         Gabriel Scherer, projet Parsifal, INRIA Saclay                 *)
+(*                                                                        *)
+(*   Copyright 2019 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+type ('a, 'b) t = Left of 'a | Right of 'b (**)
```

Comment:
Is there any reason to have `(**)`?

Reply:
There is one in the `.mli`: otherwise the following documentation comment is attached to the constructor `Right`, not to the whole type. Here there is one in the `.ml` file, and I don't think there is a reason there, that's a good point. It must come from copy-pasting from the interface file.

Reply:
I fixed it and rebased, thanks!